src/db/neo4j_driver.py

```diff
@@ -0,0 +1,18 @@
+from py2neo import Node, Relationship, Graph
+import sys
+import os
+from easydict import EasyDict
+from config import getConfig
+
+_driver = None
+
+def getClient():
+    global _driver
+    if _driver == None:
+        neo4j_config = EasyDict(EasyDict(getConfig()).db.neo4j)
+        _driver = Graph(neo4j_config.host)
+    return _driver
+      
+def query(query_sql):
+    result = getClient().run(query_sql) # return a cursor object
+    return result.data()  # transform cursor to list
```

Comment:
Use `result.data()` to transform cursor to result list, so upper layer code can use the result directly.